Chisel/CHLDuplicateDetection.m

```diff
@@ -0,0 +1,47 @@
+//
+//  CHLDuplicateDetection.m
+//  Chisel
+//
+//  Created by Greg Pstrucha on 3/1/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#if __has_feature(objc_arc)
+#error CHLDuplicateDetection.m expects ARC to be disabled.
```

Comment:
Ok I tested again, and I think it should be MRR after all. Reason for that is that object iterator will return anything that is an object, but cannot be retained. Example:

```objc
*** -[NSTaggedPointerStringCStringContainer retain] called, not supposed to happen
```